scripts/executors/src/start.ts

```diff
@@ -269,39 +280,43 @@ function createTargetDescription(
   targetName: string,
   targetsDescription: Record<string, string>,
 ) {
-  const description = targetsDescription[targetName];
+  const manualConfigDrivenDescription = targetsDescription[targetName];
   const nxTargetConfiguration = projectConfig.data.targets?.[targetName];
+  const nxMetadataDescription = nxTargetConfiguration?.metadata?.description;
+  const scriptContent: string | undefined = nxTargetConfiguration?.metadata?.scriptContent;
+  const command = scriptContent || nxTargetConfiguration?.options.command || nxTargetConfiguration?.command;
```

Comment:
scriptContent is present in projects that use npm#scripts , anything else uses nx executors ( `options.command` etc )